content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,117 @@
+---
+title: August '21 Community Gems
+date: 2021-08-31
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+picture: 2021-08-31/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/august-21-community-gems/45342t5t4egrf4w
+tags:
+  - Community
+  - Plots
+  - Pipelines
+  - CML
+  - Git
+---
+
+### [Q: Are TOML files supported for storing model metrics and displaying them via `dvc metrics show`?](https://discord.com/channels/485586884165107732/485596304961962003/865974923079319563)
+
+Thanks for the question @naeljaneLiblikas!
+
+DVC does not support TOML files for metrics. TOML files are only used for
+parameters.
+
+### [Q: How can I use object storage as a data source for MLOps with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/866393535296176148)
+
+You have a few options with DVC for this.
+
+- Set up an external cache on your object storage
+  bucket.[Check out this doc to learn more](https://dvc.org/doc/user-guide/managing-external-data#setting-up-an-external-cache)!
+- Track a large dataset in an external location without downloading it locally.
+  [This doc will tell you more](https://dvc.org/doc/command-reference/add#example-transfer-to-remote-storage).
+
+Great question @Ravi Kumar!
+
+### [Q: Is there a recommended way to specify multiple pipelines in DVC?](https://discord.com/channels/485586884165107732/485596304961962003/864230750325047316)
```

Comment:
the way the question is written here ("specify multiple pipelines in DVC") is inconsistent with the answer "specify pipelines in multiple DVC yaml files." Seems one needs to be changed.

Reply:
I think the recommendation is to keep each pipeline in a separate `dcv.yaml` file and is kind of explained in the last bullet. Maybe that needs to be moved up to the top?

Reply:
I made some updates here, so let me know what you think!

Reply:
Looks good, although you might want to clarify that multiple `dvc.yaml` files are recommended _but not required_ to specify multiple pipelines.